js/ngDialog.js

```diff
@@ -7,7 +7,7 @@
 (function (root, factory) {
     if (typeof module !== 'undefined' && module.exports) {
         // CommonJS
-        module.exports = factory(require('angular'));
+        module.exports = require('angular');
```

Comment:
@alexbeletsky That is correct. Shimming seems to be the best option. Thank you for your help!

Closing this PR as it is not correct.